scripts/Refinery.js

```diff
@@ -120,23 +121,25 @@ async function targetFACCheck(tokenAddress, amount) {
   );
 
   // Liquidity Check for targetFoundryTokken in TargetFundManager
-  const targetFoundryAssetLiquidity = await targetFoundryTokenContract.balanceOf(
-    targetFundMangerContract.address
-  );
+  const targetFoundryAssetLiquidity =
+    await targetFoundryTokenContract.balanceOf(
+      targetFundMangerContract.address
+    );
 
-  if (isTargetTokenFoundryAsset == true && Number(targetFoundryAssetLiquidity) > Number(amount)) {
-      return true;
-  }
-  else{
+  if (
+    isTargetTokenFoundryAsset == true &&
+    Number(targetFoundryAssetLiquidity) > Number(amount)
+  ) {
+    return true;
+  } else {

```

Comment:
I think else { could be removed and return false directly for code optimization.